packages/lwc-language-server/src/javascript/compiler.ts

```diff
@@ -91,7 +93,33 @@ export async function compileSource(source: string, fileName: string = 'foo.js')
             namespace: 'x',
             files: {},
         };
-        const transformerResult = await transform(source, fileName, options);
+
+        let codeTransformed = source;
+
+        if (fileName.endsWith('.ts')) {
+            try {
+                const { code } = babel.transform(source, {
```

Comment:
Lets extract this into a different method.

Reply:
Extracted.